### DISCUSSION TITLE
Configuration Settings - Suggestion

### DISCUSSION THREAD

**Discussion by sfritz85** [home-assistant/frontend#17340] [Configuration]
Would it be possible to add the following feature:

To modify the default configuration to achieve the following.  Maybe something similar to the looks of Settings > Dashboard page with ability to check/uncheck different configuration settings. 

Configuration                           Show in sidebar 
Energy                                      Admin   All users
Media                                       Admin   All users

This is what i am trying to do.  Hopefully within user interface instead of yaml.
default_config:
  exclude:
    domains:
      - energy
      - media source

This is very rough explanation but maybe someone would understand or have a better suggestion for this.

**Comment by frenck** [comment]
This is not a frontend feature request, but a backend one (YAML configuration of Home Assistant). Please use our community forums for these (hint: This feature request already exists, so if you search for it, you can upvote that one).

../Frenck
